src/data-types-lib/index.js

```diff
@@ -100,6 +101,16 @@ export const
         return true;
       }
 
+      let customValidationResult = false;
+
+      if (validate) {
+        try {
+          customValidationResult = validate(value, instance)
+        } catch (err) {
+          throw err
```

Comment:
According to the existing **validate()** pattern, custom validator errors should be appended to the resultant **errors** array.